core/src/main/resources/reference.conf

```diff
@@ -142,6 +142,10 @@ system {
   # Default number of cache read workers
   number-of-cache-read-workers = 25
 
+  # The maximum number of file hashes a single job will ask for at a time. The job will not ask for any more hashes
+  # until it receives a response to its outstanding request.
+  file-hash-batch-size = 5
```

Comment:
new config parameter, this used to be hardcoded to 100 which created scenarios where the number of I/O requests for file hashes could jump by 5000 / second as 50 jobs / sec requested 100 hashes at a time.